### DISCUSSION TITLE
ESCAM PT200 with FH8626V100 Video-Proc & SV6256P WiFi chip

### DISCUSSION THREAD

**Discussion by mybiddy** [roleoroleo/yi-hack-Allwinner-v2#683] [General]
Hi all, I could so far not find the exact corresponding YI equivalent to my ESCAM PT200 with above indicated ChipSets.
Tried loading several YI hacks but no success. Having at least an RSTP-stream to connect to my server would be great. Hints most appreciated! BR, Biddy

**Comment by roleoroleo** [comment]
Hi.
Sorry but this model is based on a different hw platform and it's not supported by this hack.
